examples/README.md

```diff
@@ -2,4 +2,35 @@
 
 Examples are typically generated within a FHIR Implementation Guide. However, the [FHIR Publisher](https://confluence.hl7.org/display/FHIR/IG+Publisher+Documentation) responsible for building Implementation Guides places additional requirements on examples that are incompatible with this IG's [data minimization criteria](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/conformance.html#data-minimization), such as requiring `id` elements and not recognizing absolute URIs like `resource:0` for within-Bundle resolution.
 
-Until these issues with the IG Publisher are resolved, we are storing our example resources in this folder outside the IG build process, and linking from the IG to these files on GitHub.
\ No newline at end of file
+Until these issues with the IG Publisher are resolved, we are storing our example resources in this folder outside the IG build process, and linking from the IG to these files on GitHub.
+
+## Scenarios:
+ - `Scenario1Bundle.json` is a patient who received two doses of Moderna vaccine, reported via CVX
+ - `Scenario2Bundle.json` is a patient who received two doses of Pfizer vaccine, reported via CVX
+ - `Scenario3Bundle.json` is a patient who tested positive on COVID19 antigen rapid immunoassay test
+
+## Examples for various code systems
+
+See `byCodeSystems` folder.
+
+| file                       | description                                      |
+|:--------------------------:|:-------------------------------------------------|
+| `air_immunization`         | Patient received one dose of AstraZeneca vaccine |
+| `atc_immunization`         | Patient received one dose of Janssen vaccine, manufacturer specified by MVX     |
+| `atc_gtin_immunization`    | Patient received one dose of Janssen vaccine, specified by ATC and GTIN systems |
+| `cvx_immunization`         | Patient received two doses of an unspecified vaccine, presumably (but not necessarily) from China, Russia, or another foriegn country |
+| `gtin_immunization`        | Patient received two doses of Moderna vaccine    |
+| `icd11_immunization`       | Patient received two doses of Pfizer vaccine, manufacturer specified by MVX     |
+| `icd11_gtin_immunization`  | Patient received two doses of Moderna vaccine, specified by ICD-11 and GTIN     |
+| `snomed_immunization`      | Patient received two doses of Pfizer vaccine, manufacturer specified by GLN     |
+| `snomed_gtin_immunization` | Patient recieved two doses of Moderna vaccine, manufacturer specified by SNOMED and GTIN |
+
+
+## Validation notes:
```

Comment:
If there are expected errors from the validator, they should be documented in the IG so implementers know to expect their tests to fail. We've been using the conformance.md page Validation section.